config/network/errors.go

```diff
@@ -0,0 +1,30 @@
+package network
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/network/p2p"
+)
+
+// ErrInvalidLimitConfig indicates the validation limit is < 0.
+type ErrInvalidLimitConfig struct {
```

Comment:
```suggestion
type InvalidLimitConfigError struct {
```
nitpick: error naming convention